ansible/templates/topo/t0.j2

```diff
@@ -116,6 +116,9 @@
         <PortChannel>
           <Name>PortChannel{{ ((index+1)|string).zfill(4) }}</Name>
           <AttachTo>{{ port_alias[28+index] }}</AttachTo>
+{% if index%2 %}
```

Comment:
Fallback is enabled only when we have odd index?
Can you please comment what is this like do?

Reply:
yes. only enable fallback on odd ports here, so we can compare the status between odd ports and even ports to make sure both are in expected state.

Thanks for the suggestion. Will add some comment here.